pkg/installmanager/installmanager.go

```diff
@@ -1689,3 +1702,48 @@ func getHomeDir() string {
 	}
 	return defaultHomeDir
 }
+
+func overrideCredentialsModeOnInstallConfig(manifestsDir, credentialsMode string) error {
+	clusterConfigPath := filepath.Join(manifestsDir, clusterConfigYAML)
+	clusterConfigBytes, err := ioutil.ReadFile(clusterConfigPath)
+	if err != nil {
+		return errors.Wrap(err, "error reading install config")
+	}
+
+	decoder := yml.NewYAMLOrJSONDecoder(bytes.NewReader(clusterConfigBytes), 4096)
```

Comment:
Curious why we're using this new lib instead of the `yaml` one already used elsewhere in this file. We know `clusterConfigBytes` is yaml, not json, right?

Reply:
My bad I tried to use the same one which we used in CCO. Replaced it with existing `yaml`